recipes/neon2sse/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+class neon2sseConan(ConanFile):
+    name = "neon2sse"
```

Comment:
The github repo is named `intel/ARM_NEON_2_x86_SSE`.
Should this package be named `ARM_NEON_2_x86_SSE`?
Or `arm-neon-2-sse`?

Reply:
Since I'm packaging this to use it in tensorflow-lite, I took the name from there
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/CMakeLists.txt#L137
But then going over the sources of the library I realized `neon2sse` is not even the right findModule.cmake name. So at this point the name I choose doesn't make much sense.
I'm open to change it.
From aesthetics I think neon2sse looks better, but it might confuse a dev looking for this package.

Reply:
Let's make it democratic since naming is always the hardest part.
arm-neon-2-x86-sse: :+1: 
arm_newon_2_x86_sse: :100: 
other: :1st_place_medal: + comment

Reply:
Totally agree on the naming part. I made the comment to discuss it asap.

🥇 I'm 50-50 split between `intel-neon2sse` and `neon2sse` (the current one).

Reply:
I prefer using intel as prefix. A quick search on Google for `NEON_2_SSE` showed me similar project names. Also if you search on Github, there are many projects using same name: https://github.com/search?q=neon_2_sse

Reply:
Do we agree on `intel-neon2sse` then?

Reply:
Okay from me!